src/main/java/org/aitesting/microservices/tripmanagement/common/models/TripInvoice.java

```diff
@@ -0,0 +1,52 @@
+package org.aitesting.microservices.tripmanagement.common.models;
+
+import java.util.Date;
+
+public class TripInvoice {
+    private String originAddress;
+    private String destinationAddress;
+    private double distance;
+    private int time;
```

Comment:
Can this be changed to "duration" to match the json fields from calculation and gmaps services?